### ISSUE TITLE
Extra indentation inside array delarations trigger errors

### ISSUE THREAD

**Issue by ToadKing** [decaffeinate/decaffeinate#131]
Coffeescript allows for extra indentation inside array and object declarations, so this should not trigger an error:

```coffee
doThing [1, 2, 3,
         4, 5, 6,
         7, 8, 9]
```

**Comment by eventualbuddha** [comment]
This works fine on master, so it'll be fixed with the next release. That might not be immediately since master regressed a bit in functionality after a major rewrite.

```
$ ./bin/decaffeinate <<EOS
> doThing [1, 2, 3,
>          4, 5, 6,
>          7, 8, 9]
> EOS
doThing([1, 2, 3,
         4, 5, 6,
         7, 8, 9]);
```

**Comment by eventualbuddha** [comment]
Fixed in v2.7.4, and probably earlier ([repl](http://decaffeinate.github.io/decaffeinate/repl/#?evaluate=false&code=doThing%20%5B1%2C%202%2C%203%2C%0A%20%20%20%20%20%20%20%20%204%2C%205%2C%206%2C%0A%20%20%20%20%20%20%20%20%207%2C%208%2C%209%5D)).
